LandonHotel.Tests/BookingServiceTests.cs

```diff
@@ -64,18 +64,20 @@ public void IsBookingCostForAdditionalPersonsCorrect()
         {
             //Arrange
             var service = Subject();
+            roomRepo.Setup(r => r.GetRoom(1)).Returns(new Room { Id = 1, Rate = 200 });
             const int guestCount = 3, rate = 200, numberDays = 2;
             System.DateTime checkin = System.DateTime.Now;
             System.DateTime checkout = checkin.AddDays(numberDays);
 
             //TODO: Calculate total booking cost using the constant values and the business rule
             //           cost per day = rate + rate * guestCount * 10/100
 
-            var totalBookingCost = 1.0; //dummy value to start
+
+            double totalBookingCost = 480; //dummy value to start
```

Comment:
Generally you would calculate this with the constant values so that you can parameterize the process